scalableminds/webknossos
# reserveManualUpload Datastore Route

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- with the token of a team manager, and the id of a folder that team has access to, run `curl -X 'POST' http://localhost:9000/data/datasets/reserveManualUpload?token=<token> --data-raw '{"datasetName":"newDatasetName","organization":"sample_organization","initialTeamIds":["662a26b0d60000480b720d00"],"folderId":"662a26a3d60000480b720cff"}'  -H 'content-type: application/json'`
- Place the dataset on disk
- The disk scan should scrape up that dataset after at most one minute. Alternatively, call `POST /triggers/reload/:organizationName/:datasetName?token=xyz`
- Then on a page refresh, the dataset should be available and the team manager should be able to edit its settings
- if the user is not team manager, or attemts this with a folder that their managed team does not have access to, the token authentication should fail.

### Issues:
- fixes #7457 

------
